File: DayOne/Python/LevelOne/perfect_square_array.py
import math

def perfect_square_check(array):

    new_array = []

    for index_count in range(len(array)):

        number_square_root = math.sqrt(array[index_count])

        if number_square_root % 1 == 0:

            new_array.append(array[index_count])

    return new_array

File: DayOne/Java/LevelOne/PerfectSquareArray.java
public class PerfectSquareArray{

    public static double [] perfectSquareCheck(int [] array){

    int perfectSquareCount = 0;

    for (int indexCount = 0; indexCount < array.length; indexCount++){

        double numberSquareRoot = Math.sqrt(array[indexCount]);

        if (numberSquareRoot % 1 == 0) perfectSquareCount++;

        }

    double [] newArray = new double [perfectSquareCount];

    int index = 0;

    for (int newArrayIndexCount = 0; newArrayIndexCount < array.length; newArrayIndexCount++){

        double numberSquareRoot = Math.sqrt(array[newArrayIndexCount]);

        if (numberSquareRoot % 1 == 0) {
            
            newArray[index] = array[newArrayIndexCount];
        
            index++;
        
        }
    
    } 

    return newArray;
    }
}
